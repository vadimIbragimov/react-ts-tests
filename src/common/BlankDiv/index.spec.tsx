import React from 'react';
import { mount, render, shallow } from 'enzyme';
import { BlankDiv } from './index';

describe('BlankDiv component', () => {
  it('should slhallow BlankDiv component', () => {
    const component = shallow(<BlankDiv />);
    expect(component).toMatchSnapshot();
  });
  it('should mount BlankDiv component', () => {
    const component = mount(<BlankDiv />);
    expect(component).toMatchSnapshot();
  });
  it('should render BlankDiv component', () => {
    const component = render(<BlankDiv />);
    expect(component).toMatchSnapshot();
  });
  it('should contain text "qwe"', () => {
    const component = shallow(<BlankDiv />);
    expect(component.text()).toEqual('qwe')
  })
});
