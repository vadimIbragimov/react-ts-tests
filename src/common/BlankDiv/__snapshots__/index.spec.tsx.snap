// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlankDiv component should mount BlankDiv component 1`] = `
<BlankDiv>
  <div
    className="blank-div"
  >
    qwe
  </div>
</BlankDiv>
`;

exports[`BlankDiv component should render BlankDiv component 1`] = `
<div
  class="blank-div"
>
  qwe
</div>
`;

exports[`BlankDiv component should slhallow BlankDiv component 1`] = `
<div
  className="blank-div"
>
  qwe
</div>
`;
